src/tokenserver/extractors.rs

```diff
@@ -62,44 +109,162 @@ impl FromRequest for TokenserverRequest {
     }
 }
 
+impl FromRequest for Box<dyn Db> {
```

Comment:
This is different from what's done for syncstorage [here](https://github.com/mozilla-services/syncstorage-rs/blob/master/src/db/transaction.rs#L203). Instead of implementing `FromRequest` for the DB pool, I just implement it for the DB connection itself. I think this simplifies the usage of the database, and I couldn't see a downside in this particular circumstance